app/views/pages/components/categories-listing/listed-category.ejs

```diff
@@ -2,6 +2,7 @@
 <a href='/?category=<%=category.key%>' class='listed-category'>
     <%- include(`../../../ssrUtils/responsiveImage`, {
         url: `${category.imageUrl}`,
+        version: 1,
```

Comment:
Why is it hardcoded as 1

Reply:
the endpoint requires a version, I can remove the requirement if you want